src/libstd/path.rs

```diff
@@ -1520,6 +1530,9 @@ impl<'a> From<Cow<'a, Path>> for PathBuf {
 
 #[stable(feature = "shared_from_slice2", since = "1.24.0")]
 impl From<PathBuf> for Arc<Path> {
+    /// Converts a `PathBuf` into a `Arc<Path>`.
+    /// This conversion happens in place.
+    /// This conversion does not allocate memory.
```

Comment:
Because this one is logically three parts, you'll need two blank lines to separate them out here